### DISCUSSION TITLE
既然已经5.0了，配置是不是稳定了呢

### DISCUSSION THREAD

**Discussion by christaikobo** [IrineSistiana/mosdns#580] [Ideas]
个人觉得每几个月就大版本更新导致配置文件不通用，其实并不利于本项目的推广

**Comment by sjtuross** [comment]
配置改来改去真心吃不消

**Comment by luzhaohui** [comment]
v4.5.3 用着，不搞了，每个大版本配置文件都不通用，学习成本太高了。

**Comment by christaikobo** [comment]
@IrineSistiana 请问作者啥想法呀
另，建议保留旧版本的wiki
好多人还停留在V3V4，查WIKI都查不到

**Comment by zqp561022** [comment]
换版本就换配置，确实学习成本高
